core/pom.xml

```diff
@@ -33,6 +33,11 @@
   <name>Spark Project Core</name>
   <url>http://spark.apache.org/</url>
   <dependencies>
+    <dependency>
+      <groupId>com.thoughtworks.paranamer</groupId>
```

Comment:
I see, so we need to make sure to include paranamer, as it won't work if it inherits an older one at runtime? This is OK. I would change this slightly; you don't need to include `<version>` here as that's managed in the parent POM. And in the parent, you can declare this as `<scope>runtime</scope>` I suppose, because it's not directly used, but it won't matter much.

Reply:
Thanks. Of course, I'll update like that.